### DISCUSSION TITLE
CSS ile sarkan menü nasıl yaparım

### DISCUSSION THREAD

**Discussion by ademilter** [ademilter/homepage#44] [Q&A] [answered]
Hayırlı akşamlar Adem abi,

genişlik atıyorum 800px iken kırmızı dairedeki menü diğer menüsünün altına iniyor, sonra ekran biraz daha genişlik kazanırsa kırmızı dairedeki menü bu sefer sol taraftaki sıralı yerine geliyor bunu nasıl yapacağım kafamda oturtamadım yardımcı olabilirsen çok sevinirim.

javasscriptle sürekli genişlik kontrolü yaparak yapmayı deniyorum ama pek mantklı gelmiyor.

![untitled-1](https://user-images.githubusercontent.com/187922/27519318-99f74614-59f9-11e7-955a-3a99e8e3d1ec.jpg)

**Comment by ademilter** [comment]
Böyle bir menü için js yazmaya gerek yok. Aşağıdaki örnekte mantığı anlatmaya çalıştım
https://codepen.io/ademilter/pen/KqXqzq?editors=1100

**Comment by kaanklky** [comment]
Hocam arkadasin bahsettigi seyi yanlis anlamissiniz. Menunun en sonunda bir dropdown olacak, viewport genisligi dustukce dropdowndan bir onceki element dropdowna gececek.

**Comment by ademilter** [comment]
@kaanklky sen yazdıktan sonra olayı anladım. hızlıca okuyup olayı tam anlamamışım başta 🐴
yapılmak istenilen bu sanırım https://www.youtube.com/watch?v=lrnMcOV_VxA

bunu yapan jquery pluginler vardı. bulursam yazıcam.

**Comment by abdullah** [comment, marked as answer]
https://github.com/VPenkov/okayNav/blob/master/README.md

Ehehe bu var

**Comment by ademilter** [comment]
@abdullah dünden beri bunu arıyorum abi :))) eyv

**Comment by berkaynizam** [comment]
http://gijsroge.github.io/priority-nav.js/
Alternatif olarak :)

**Comment by abdullah** [comment]
Ya ben arada sirada bakiyorudum buraya :D
